app/javascript/controllers/language_change_controller.js

```diff
@@ -1,10 +1,16 @@
 import { Controller } from "@hotwired/stimulus"
+import { Turbo } from "@hotwired/turbo-rails";
+import { getCookie } from "../mixins/cookie";
 
 // Connects to data-controller="language-change"
 export default class extends Controller {
 
-  dispatchLangChangeEvent() {
+  connect() {
+    const locale = getCookie('locale');
+    document.querySelector(`#language-select option[value="${locale}"]`)?.selected = true;
```

Comment:
this line does not work, because of the `?`

Reply:
works fine for me, this syntaxe is called `nullsafe operator` so if `document.querySelector(`#language-select option[value="${locale}"]`)` return `null` it will not continue